docs/unit2.mdx

```diff
@@ -1,7 +1,10 @@
 ---
 sidebar_position: 2
+sidebar_label: "Unit 2: Touching the Terminal"
```

Comment:
This should be autogenerated from the markdown, so I don't think it's necessary. I added it to unit 1 because I wanted the sidebar to say "Unit 1: GitHub Setup" instead of the longer "Unit 1: Git-ing GitHub Setup"

```suggestion
```

Reply:
Without the sidebar label, it would use the name of the file instead. 
![image](https://user-images.githubusercontent.com/9029237/161649214-cc64c55f-0f2a-428c-a84c-19636c7621ce.png)

Reply:
Oop ok good call!